lib/src/models/rover/controller.dart

```diff
@@ -62,7 +77,7 @@ class Controller extends Model {
 
 	/// Sends a command over the network or over Serial.
 	Future<void> sendMessage(Message message) async {
-		if (models.serial.isConnected) {
+		if(models.serial.isConnected && (models.serial.connectedDevice == teensyCommands[message.messageName])){
```

Comment:
> ```dart
> ... && (models.serial.connectedDevice == teensyCommands[message.messageName])
> ```
1) This is a "gotcha" bug -- there's a `Serial` service, and there's a `SerialModel`. They both have a field called `connectedDevice`, but the model's idea of a device is something like `COM13`, while the service uses the `Device` enum. So this should be `services.serial.connectedDevice`, not `models.serial`. 
2) Related, this won't work because `teensyCommands` has String keys but `connectedDevice` is a `Device`. Change the `teensyCommands` map to make that work,

Reply:
fixed that but the new protobuf files haven't been compiled in this branch so that map which includes Device.ARM and others are not found